packages/governance/src/contractGovernance/governParam.js

```diff
@@ -72,29 +85,32 @@ const setupParamGovernance = async (
   /** @type {WeakSet<Instance>} */
   const voteCounters = new WeakSet();
 
-  /** @type {VoteOnParamChange} */
-  const voteOnParamChange = async (
-    paramSpec,
-    proposedValue,
+  /** @type {VoteOnParamChanges} */
+  const voteOnParamChanges = async (
```

Comment:
"vote on changes" doesn't convey that it's atomic. consider "changeset" as the term for a collection of param changes that live or die to together.